homeassistant/components/simplisafe/__init__.py

```diff
@@ -376,6 +376,7 @@ async def async_update_options(hass, config_entry):
     """Handle an options update."""
     simplisafe = hass.data[DOMAIN][DATA_CLIENT][config_entry.entry_id]
     simplisafe.options = config_entry.options
```

Comment:
This is no longer necessary because you are reloading the config entry, so this object will be unloaded.